### PR TITLE
Deployment/add all servers to script

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: scripts/configure.sh [hunk 1]
```diff
@@ -1,31 +1,33 @@
 #!/usr/bin/env bash
 
 #Archive our most recent deployment
-cd /opt/www/
+DEPLOYMENT_DIR='/opt/www/'
+cd $DEPLOYMENT_DIR
 APPLICATION_DIR='oneyou_haytool'
-APPLICATION_EXISTS=`ls $APPLICATION_DIR`
+APPLICATION_EXISTS=`sudo ls $APPLICATION_DIR`
```
Comment (eddiejaoude) [inline scripts/configure.sh]: Really needs `sudo`?

File: scripts/configure.sh [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
 cd $APPLICATION_DIR/
 
 echo "Preparing cache"
-`rm -rf app/cache/*`
-`rm -rf app/logs/*`
+`sudo rm -rf app/cache/*`
+`sudo rm -rf app/logs/*`
 
 `php app/console cache:clear --env=prod`
+`php app/console doctrine:migrations:migrate --no-interaction  > /dev/null 2>&1`
```
Comment (eddiejaoude) [inline scripts/configure.sh]: If fails, will not see error?

File: scripts/configure.sh (hunk 1 above)
Comment (brettminnie) [inline scripts/configure.sh]: I do due to security settings, the dir is locked down to the scp user, this
user has to run as sudo to w/x

File: scripts/configure.sh (hunk 2 above)
Comment (brettminnie) [inline scripts/configure.sh]: Not finished yet, this is to test manually

I'm going to change the script once it all works to redirect all output to
a log

Comment (eddiejaoude) [inline scripts/configure.sh]: :+1:

**Pull request merged**